src/python/pants/core/goals/lint.py

```diff
@@ -193,6 +193,18 @@ def activated(cls, union_membership: UnionMembership) -> bool:
         "--only",
         help=only_option_help("lint", "linter", "flake8", "shellcheck"),
     )
+    skip_formatters = BoolOption(
+        "--skip-formatters",
+        default=False,
+        help=softwrap(
+            f"""
+            If true, skip running all formatters in check-only mode.
```

Comment:
Can we just call it "lint mode"?

Reply:
Perhaps? I think `lint` is generally not super well understood and it might be a tautology? I'm thinking `check-only` mode may be clearer.

Reply:
My only heartburn comes from the fact that Pants has a `check` goal. `check-if-changed` mode?

turbo minor feel free to drop :)